link/src/main/java/com/stripe/android/link/ui/inline/InlineSignupViewModel.kt

```diff
@@ -32,17 +30,18 @@ internal class InlineSignupViewModel @Inject constructor(
     private val prefilledEmail =
         if (linkAccountManager.hasUserLoggedOut(customerEmail)) null else customerEmail
 
-    val emailElement: SectionFieldElement =
-        EmailSpec.transform(mapOf(IdentifierSpec.Email to prefilledEmail))
+    val emailController = SimpleTextFieldController.createEmailSectionController(
+        prefilledEmail
+    )
 
     /**
      * Emits the email entered in the form if valid, null otherwise.
      */
     private val consumerEmail: StateFlow<String?> =
-        emailElement.getFormFieldValueFlow().map { formFieldsList ->
+        emailController.formFieldValue.map {
             // formFieldsList contains only one element, for the email. Take the second value of
             // the pair, which is the FormFieldEntry containing the value entered by the user.
```

Comment:
Can delete the comments now.

Reply:
resolved in this PR: https://github.com/stripe/stripe-android/pull/4992